src/models/index.d.ts

```diff
@@ -62,4 +18,12 @@ interface IInvokeConfig {
   functions: { testResults: { name: string }; techRecords: { name: string; mock: string }; getActivities: { name: string } };
 }
 
-export { ISPConfig, IActivity, IInvokeConfig, ITestType, ITestResults, IActivitiesList };
+interface IActivityParam {
+  testerStaffId: string
```

Comment:
Can you correct the end of each in this interface please, should end with a semi-colon.